src/main/java/core/basesyntax/service/impl/DailyReportWriterImpl.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.DailyReportWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class DailyReportWriterImpl implements DailyReportWriter {
+    private final StorageDao storageDao;
+
+    public DailyReportWriterImpl(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
+
+    @Override
+    public void write(String toFilePath) {
```

Comment:
This method responsible for creating a report and writing it to file, this violates SRP, lets crate 2 separate services for this